src/engine/GroupBy.h

```diff
@@ -506,3 +507,8 @@ class GroupBy : public Operation {
   // TODO<joka921> Also inform the query planner (via the cost estimate)
   // that the optimization can be done.
 };
+
+// _____________________________________________________________________________
+template <typename A>
+concept SupportedAggregates =
+    ad_utility::SameAsAnyTypeIn<A, GroupBy::AggregationDataVectors>;
```

Comment:
As concepts can be inside classes, they should at least be inside a `namespace`,
or (if only used inside a single .cpp file) be defined in that cpp file directly.